.coveragerc

```diff
@@ -0,0 +1,3 @@
+[run]
+omit =
+  mailinglogger/tests/*.py
```

Comment:
No, we want tests included. Measuring coverage on test files is an excellent way to check they're actually being run.
This has caught me out several times, so please remove this file.